Makefile

```diff
@@ -66,6 +66,7 @@ deploy: $(ARTIFACTS_DIR)/kubeconfig.yaml get-base-domain
 
 # Get kubernetes context
 $(ARTIFACTS_DIR)/kubeconfig.yaml: $(ARTIFACTS_DIR)/terraform.tfstate get-base-domain
+	if [ ! -d $(ARTIFACTS_DIR) ]; then mkdir $(ARTIFACTS_DIR); fi
```

Comment:
This one should not be necessary

Reply:
This one is necessary, I had exactly the same issue there

Reply:
This is weird, it should have been created in the target in dependence. And anyway if it does not exist yet at this step, I guess nothing will work. I have to take a closer look at this.